modules/installation-special-config-raid.adoc

```diff
@@ -87,19 +99,12 @@ storage:
       with_mount_unit: true
 ----
 
-. Run Butane to create a RAID manifest from the Butane config you created in the previous step, for example:
-+
-[source,terminal]
-----
-$ butane $HOME/clusterconfig/<butane_file> -o ./<manifest_name>.yaml <1>
-----
-<1> Replace `<butane_file>` and `<manifest_name>` with the file names from the previous step. For example, `raid1-alt-storage.bu` and `raid1-alt-storage.yaml` for secondary disks.
-
-. Apply the manifest to your cluster by running the following command:
+. Create a RAID manifest from the Butane config you created in the previous step and save it to the `<installation_directory>/openshift` directory. For example, to create a manifest for the compute nodes, run the following command:
 +
 [source,terminal]
 ----
-$ oc create -f <manifest_name>.yaml
+$ butane $HOME/clusterconfig/<butane_config>.bu -o <installation_directory>/openshift/<manifest_name>.yaml <1>
 ----
+<1> Replace `<butane_config>` and `<manifest_name>` with the file names from the previous step. For example, `raid1-alt-storage.bu` and `raid1-alt-storage.yaml` for secondary disks.
 
 . Save the Butane config in case you need to update the manifest in the future.
```

Comment:
In some similar procedures we say e.g. `Continue with the remainder of the {product-title} installation.` to help make it clear that this needs to be done at installation time.